### PR TITLE
PR Fix2 for Aeotec Heavy Duty Switch

### PR DESCRIPTION
**Pull request by jacobvoegele**

### CODE REVIEW

File: drivers/SmartThings/zwave-switch/src/init.lua [hunk 1]
```diff
@@ -161,7 +161,8 @@ local driver_template = {
     lazy_load_if_possible("multi-metering-switch"),
     lazy_load_if_possible("zooz-zen-30-dimmer-relay"),
     lazy_load_if_possible("multichannel-device"),
-    lazy_load_if_possible("aeotec-smart-switch")
+    lazy_load_if_possible("aeotec-smart-switch"),
+    lazy_load_if_possible("aeotec-aeotec-heavy-duty")
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-switch/src/init.lua:165]:
```suggestion
    lazy_load_if_possible("aeotec-heavy-duty")
```

File: drivers/SmartThings/zwave-switch/fingerprints.yml [hunk 2]
```diff
@@ -784,11 +784,11 @@ zwaveManufacturer:
     productType: 0x1801
     productId: 0x1000
     deviceProfileName: metering-switch
-  - id: 0086/004E
-    deviceLabel: Aeotec Switch
+  - id: "Aeotec/Heavy Duty"
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-switch/fingerprints.yml:787]: I may be remembering this wrong but I thought we couldn't update the ID of an existing fingerprint. @greens ?

**Comment by lelandblue** [comment]
@greens Could you review this PR? 

This has passed WWST Testing, so I believe I can Merge this PR 1325 and close https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1109.

**Review by greens — APPROVED** [review]
Looks good. I tested Tom's caveat and it doesn't _seem_ to be an issue, but we should still monitor the alpha deploy.

**Pull request merged**